lib/igniter/project/application.ex

```diff
@@ -89,18 +89,17 @@ defmodule Igniter.Project.Application do
       |> Sourceror.Zipper.zip()
 
     with {:ok, zipper} <- Igniter.Code.Function.move_to_def(zipper, :application, 0),
-         zipper <- Igniter.Code.Common.rightmost(zipper),
+         {:ok, zipper} <- Igniter.Code.Common.move_right(zipper, &Igniter.Code.List.list?/1),
```

Comment:
I think we still need `rightmost` here.

Reply:
That would be the "thing that this function returns"

Reply:
Well either there's a bug in rightmost or thats not what happens. This was to handle the edge case where there was code before the keyword list. `rightmost/1` wasn't handling that

Reply:
🤔 okay, that will have to be addressed in that case. What this should mean is that if you did this:

```elixir
def application() do
  []

  [
    application: {Module, []}
  ]
end
```

that test would fail (because the current implementation stops at the first list). So the bug you're referring to will have to be fixed some other way. I can try to find some time today to fix the bug w/ `rightmost`, if you wouldn't mind pushing it back up to use `rightmost`.

Reply:
I added a test to verify https://github.com/ash-project/igniter/pull/128